### ISSUE TITLE
ETA for ranges in ChartkickCard?

### ISSUE THREAD

**Issue by yrashk** [avo-hq/avo#772]
First of all, thanks for releasing the 2.0 series!

I just wanted to see, what's the plan for this to-do item, do you think it'll be addressed soon?

https://github.com/avo-hq/avo/blob/2b39367be48ccc31b1415bc2b8897f530f01a014/spec/dummy/app/avo/cards/example_area_chart.rb#L9

**Comment by adrianthedev** [comment]
Haha. You dug through the code and found that. Nice!
I'll probably work on it next week. 

Have you used the dashboards feature and felt this lacking?

**Comment by yrashk** [comment]
Yeah. It'd be nice to have in my dashboard to see differently ranged charts.

**Comment by adrianthedev** [comment]
Hey @yrashk. Apologies for the long wait. The last few weeks have been a bit loaded for me.

I enabled the ranges on all types of cards. Let me know if you see anything "wonky" with them.

FYI, I changed the way `ranges` were processed by Avo. Before we'd take the array and figure out if there were integers and cast them all to "days" (`4` becomes `4 days`) and that doesn't really work when you don't need days. So Avo just takes that option `range` and passes it to `options_for_select`. This way it's more Railsy. 

[Upgrade guide](https://docs.avohq.io/2.0/upgrade.html#upgrade-from-2-7-to-2-8)
